scrapy/utils/conf.py

```diff
@@ -159,7 +145,7 @@ def feed_complete_default_values_from_settings(
 def feed_process_params_from_cli(
     settings: BaseSettings,
     output: list[str],
-    output_format: str | None = None,
+    *,
```

Comment:
Same as with `build_component_list` here. I am OK with this approach, but I would see the argument for deprecating the existence of the parameter separately in a more robust way.